src/main/java/games/strategy/triplea/attachments/UnitAttachment.java

```diff
@@ -3364,6 +3337,12 @@ public String toStringShortAndOnlyImportantDifferences(final PlayerID player, fi
     return stats.toString();
   }
 
+  private String joinRequiredUnits(final List<String[]> units) {
```

Comment:
Good move extracting this method.

1. Note that it can be static.
1. Might consider adding some unit tests since it was extracted.

Reply:
It's really just a three-liner, and I don't think it'll be worth adding a test for this unless we move it to a utils class and make it public